### PR TITLE
Replace outdated ledgers in hardfork packages

### PR DESCRIPTION
**Pull request by nholland94**

### CODE REVIEW

**Comment by nholland94** [comment]
!ci-build-me

File: scripts/create_hardfork_deb.sh [hunk 1]
```diff
@@ -47,6 +47,10 @@ for ledger_tarball in $LEDGER_TARBALLS; do
   cp "${ledger_tarball}" "${BUILDDIR}/var/lib/coda/"
 done
 
+# Overwrite outdated ledgers that are being updated by the hardfork
+cp "${RUNTIME_CONFIG_JSON}" "${BUILDDIR}/var/lib/coda/${NETWORK_NAME}.json"
```
[the remark this answers is not in the document]
Comment (georgeee) [reply scripts/create_hardfork_deb.sh:51]: Maybe we could consider moving old ledger (if any) to `{network-name}.old.json`?
And updating verification script accordingly.

Of course, if my concern is valid (not sure about it).

**Comment by deepthiskumar** [comment]
Nightly CI https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/1846

File: scripts/create_hardfork_deb.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (nholland94) [reply scripts/create_hardfork_deb.sh:51]: I can do it just to be safe, doesn't seem harmful.

**Comment by nholland94** [comment]
!ci-build-me

**Comment by nholland94** [comment]
I tested the build here https://buildkite.com/o-1-labs-2/hardfork-package-generation/builds/226#018eee28-bc18-4234-9110-dd975fc9304d. Everything looks as expected.

**Pull request merged**